components/landing/LandingCta.vue

```diff
@@ -75,5 +78,16 @@ export default class extends Vue {
     margin-left: $spacing-06;
     fill: currentColor;
   }
+
+  svg {
```

Comment:
Use the same strategy here. Even better, consider addressing issue #916 first, in a separated PR, before continuing with this one, and you should be able to reuse everything in the other component for free.